### PR TITLE
clean: factor test_expr_min_max out of test_common

### PR DESCRIPTION
**Pull request by condekind**
## What type of PR is this? (check all applicable)

- [x] 💾 Refactor
- [ ] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 🔧 Optimization
- [ ] 📝 Documentation
- [x] ✅ Test
- [ ] 🐳 Other

## Related issues 

- Related issue #401 

## Checklist

- [x] Code follows style guide (ruff)
- [ ] Tests added 
- [ ] Documented the changes

### CODE REVIEW

**Comment by condekind** [comment]
Strange, some unrelated tests failed :thinking:

File: tests/expr/min_max_test.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from typing import Any
+
+import narwhals as nw
```
Comment (MarcoGorelli) [inline tests/expr/min_max_test.py:3]:
```suggestion
import narwhals.stable.v1 as nw
```
not unrelated 😉

**Review by FBruzzesi — COMMENTED** [review]
Thanks for the PR! I hope the comment explains why CI is failing :)

File: tests/expr/min_max_test.py (hunk 1 above)
Comment (condekind) [reply tests/expr/min_max_test.py:3]: Oh my bad! Thanks for the quick reply!

[the remark this answers is not in the document]
Comment (condekind) [reply tests/expr/min_max_test.py:3]: Thank you for clarifying it, plus the quick reply!

**Comment by condekind** [comment]
Oh I committed manually, is that a problem? I didn't use GH's interface, I hope that's not a problem, it was just a small change

**Comment by MarcoGorelli** [comment]
ah sorry i hadn't noticed at first, but this overlaps with https://github.com/narwhals-dev/narwhals/pull/457

**Comment by condekind** [comment]
Alright, closing in favor of #457

**Pull request closed without merging**